test/functional/apps/dashboard/_dashboard_filtering.js

```diff
@@ -101,6 +101,63 @@ export default function ({ getService, getPageObjects }) {
       });
     });
 
+    describe.skip('using a pinned filter that excludes all data', async () => {
```

Comment:
Skipping because this hits the same flakiness as the below?

Reply:
becouse its dependand on the skipped test